data/idk.csv

```diff
@@ -0,0 +1,106 @@
+question,instructions,query,db_name,query_category,k_shot_prompt

```

Comment:
Should we just remove the k_shot_prompt col since it's empty for this anw?

Reply:
good point, done!